cmd/generate-release/scripts/setup-kcc.bash

```diff
@@ -79,7 +80,10 @@ spec:
 EOF
 
 # XXX: Give everything a moment to be created.
-sleep 30
+sleep 180
```

Comment:
nit: is the sleep 180 needed -- can we increase the retry you added below to something large instead i.e. 60 retries w/10 second sleep (10 minutes total wait)?